datasets/us/doh_debarred/us_doh_debarred.yml

```diff
@@ -0,0 +1,55 @@
+name: us_doh_debarred
+title: Debarred people and companies from us department of health
+entry_point: crawler.py
+coverage:
+  frequency: daily
+  start: "2024-03-13"
+summary: >
+  A list of debarred peoples and companies listed in US Department of Health and Human Service.
+description: |
+  This dataset provides information on Individuals and entities debarred from participation in Medicare,
+  Medicaid, and all other Federal health care programs for various reasons including conviction for Medicare 
+  or Medicaid fraud.
+  Debarments are imposed under the authority of sections 1128 and 1156 of the Social Security Act.
```

Comment:
let's use the word Excluded/exclusions rather than debarred/debarments to be consistent with their terminology. In the topics we're limited to the topics defined in our data model, but here it's best to align with their wording.